app/models/passwordless/session.rb

```diff
@@ -13,11 +13,13 @@ class Session < ApplicationRecord
       :expires_at,
       :user_agent,
       :remote_addr,
-      :token,
+      :token_digest,
```

Comment:
I thought calling the database column `token_digest` made it more clear that it's not the actual token. The tradeoff is this is a breaking change and assumedly would require a major version bump and running a migration. We can keep the column name the same to make upgrading easier, but you could still call it a breaking change if there are any valid non-hashed tokens in the database when you upgrade.